.github/workflows/github-actions-build.yml

```diff
@@ -6,7 +6,6 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - 16.x
```

Comment:
Could you please move this to a separate PR? That will make more sense in the history.

Reply:
Of course. Will do.